components/eam/docs/tech-guide/index.md

```diff
@@ -1 +1,45 @@
-start of the EAM Technical Guide
+This Technical Guide describes the physics of EAM.
+
+# Dynamics
+
+- [HOMME](homme.md): The dynamical core used in EAMv3.
+
+# Physics
+
+## Microphysics
+
+- [P3](p3.md): The stratiform cloud microphysics scheme in EAMv3.
+
+## Shallow convection / turbulence
+
+- [CLUBB](clubb.md): The parameterization of subgrid-scale turbulence and clouds in EAMv3.
+
+## Deep convection
+
+- [Zhang-McFarlane](zm.md): The deep convection parameterization in EAMv3.
+
+## Radiation
+
+- [RRTMG](rrtmg.md): The parameterization of radiation in EAMv3.
+
+## Aerosol
+
+- [MAM4](mam4.md): The primary parameterization scheme of aerosols in EAMv3.  
```

Comment:
Thanks @keziming. I had intended MAM5 to describe the addition of the fifth mode for the stratospheric coarse mode and the MAM4 to describe all of the aerosol processes that we have inherited from EAMv0. Since MAM4 that we inherited covers most of the processes that we run in EAMv3, I had noted it as the primary parameterization scheme. 
Maybe there is a better way to describe what is under MAM4 and MAM5. @hwangacme - do you have any thoughts?

Reply:
Thanks @keziming for pointing this out. As I discussed with @crterai offline, a higher-level, explicit description of MAM5 as the default option of EAMv3 aerosol module is more preferable.

Reply:
Thanks @hwangacme. I'll make that change.